rfcs/077-enriching-addressables-with-works/README.md

```diff
@@ -0,0 +1,161 @@
+# RFC 077: Content API: Enriching addressable content types with Works data
+
+We'd like to enrich the content pages of our website, i.e. those whose data is retrieved from Prismic, with previews of Works that are linked to within them. This RFC proposes augmenting the [Addressable content types](https://github.com/wellcomecollection/docs/tree/main/rfcs/062-content-api-all-search#addressable-content-types) with Works data and creating a new endpoint for retrieving individual Addressable items, so that we can create the desired UI with a single API call.
```

Comment:
Could be worth adding context for what "Addressable" means (maybe [this](https://github.com/wellcomecollection/docs/tree/ea2b796ea49d016039990fa6c4735d9ff1c4314d/rfcs/062-content-api-all-search#addressable-content-types)?)

Reply:
'Addressable content types' links to that description already. Would you rather I remove the link and write it here instead?

Reply:
🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️ my brain ignored the link, IGNORE ME.